### ISSUE TITLE
Style=Call or something similar for INTJ specifically to get someone's attention

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/docs#1071]
I see a few annotations for types of `INTJ`, especially `Style=Expr` for `hmm` although I think it's a little ill defined

One type we were looking to add to `INTJ` for Sindhi is that of getting a specific person's attention, such as `Hey_INTJ AngledLuffa, ...`  There is no existing `Style=` feature for such an `INTJ`, and I don't see any other feature labels which would fill that role. @muteeurahman 

Does this seem like a useful addition?  Would it be something applicable to other languages as well?  I personally could see it having a place in English and would be able to help annotate it there

**Comment by nschneid** [comment]
Is there a morphological class that has that meaning? Or is it just part of the lexical meaning?

I have always understood `Style` to be for non-default forms of words, where an alternate spelling indicates something about the speaker's attitude. I would say "hmm" is the default spelling (and lemma) and "hmmm" etc. is a lengthened form that could indicate expressiveness.

**Comment by nschneid** [comment]
(Opened UniversalDependencies/UD_English-EWT#554 regarding possible overuse of Style in English corpora)

**Comment by AngledLuffa** [comment]
Ok, that's fair.  So your suggestion would be to simply not have any feature that indicates that?  Or we could add an IntjType which may or may not cover the "hey you" usage in other languages as well

**Comment by dan-zeman** [comment]
In one of the Indian langs I've seen `Case=Voc`. It sounds a bit odd to me but I think in these languages vocative is traditionally recognized since Sanskrit, and it's often shown with an interjection (possibly in addition to a nominal suffix). I'm not sure I support it on `INTJ` though...

**Comment by nschneid** [comment]
If adding an IntjType feature, what would the other values be? Would the goal be to partition the set of all INTJ lexemes?

**Comment by amir-zeldes** [comment]
I agree this isn't so much a style as a function, and normally this would be the child of a deprel `vocative` token (though if the vocative is the root, that is no longer visible in the deprel). Making some kind of `IntjType` is an option, but I agree with Nathan, that would suggest looking at all INTJ types and making some kind of taxonomy.

**Comment by dan-zeman** [comment]
> normally this would be the child of a deprel `vocative` token

I don't think so. If the sentence is something like

* _Hey Joe, come over here_

then we will have `vocative(come, Joe)`, not `vocative(come, Hey)`. I would either do `discourse(come, Hey)` as with other interjections, or, if we go for the interpretation that _Hey_ brings the `Case=Voc` feature, we could do `case(Joe, Hey)`. I now realize why it sounds familiar to me. I was recently revisiting the validation tests for DEPREL-UPOS compatibility, and I saw that `INTJ` was allowed for `case`, with this comment:

> Interjection can also act as case marker for vocative, as in Sanskrit: भोः भगवन् / bhoḥ bhagavan / oh sir.

**Comment by AngledLuffa** [comment]
Excellent, thanks for the discussion.  We'll remove the feature from the proposed Sindhi dataset.

**Comment by amir-zeldes** [comment]
> then we will have vocative(come, Joe), not vocative(come, Hey).

I think my comment may have been hard to parse - by "this would be the child of a deprel vocative token" I meant that "hey" might be the child of a (different) token, whose deprel from its parent (the grandparent) is "vocative" (in your example, that grandparent is "come"). So I think we are saying the same thing 😅

**Comment by rueter** [comment]
> > normally this would be the child of a deprel `vocative` token
> 
> I don't think so. If the sentence is something like
> 
>     * _Hey Joe, come over here_
> 
> 
> then we will have `vocative(come, Joe)`, not `vocative(come, Hey)`. I would either do `discourse(come, Hey)` as with other interjections, or, if we go for the interpretation that _Hey_ brings the `Case=Voc` feature, we could do `case(Joe, Hey)`. I now realize why it sounds familiar to me. I was recently revisiting the validation tests for DEPREL-UPOS compatibility, and I saw that `INTJ` was allowed for `case`, with this comment:
> 
> > Interjection can also act as case marker for vocative, as in Sanskrit: भोः भगवन् / bhoḥ bhagavan / oh sir.

In the definition of `case`, [https://universaldependencies.org/u/dep/all.html#al-u-dep/case](https://universaldependencies.org/u/dep/all.html#al-u-dep/case),  it is stressed that _Case-marking elements are treated as dependents of the noun they attach to or introduce. (Thus, contrary to SD, UD abandons treating a preposition as a mediator between a modified word and its object.) The case relation aims at providing a more uniform analysis of nominal elements, prepositions and case in morphologically rich languages_

In light of this we can re-examine  _Hey, Joe, come over here_
The word “Hey” is used to capture someone's attention -- at least in the Salish Sea region. More specifically, it is used to capture the attention of “Joe”, whom we subsequently tell to move himself in the direction of the speaker.

(1) Let us assume that the use of commas is a matter tradition in English punctuation rules, where the addressee is set off with commas;
(2) There is no other indication in the written source that the proper noun «Joe» has undergone form and become a vocative;
(3) The word «Hey», however, is collocated with the second person, addressee, or addressee-candidate, but not the first or third person, e.g., *hey, me..., *hey, us..., *hey, him..., *hey, her..., *hey, it..., *hey, them...

This function of activating or eliciting “addresseehood” would seem to parallel the function referenced by the term “vocative”.

One might consider the similarity of words, such as “hello” or “dear”. 

* These two words, however, are salutary, whereas “Hey” is not. 
* The words “hello” and “dear” can appear with first, second and third person, whereas the word “hey” avoids first and third person.
* The word “hey”, without an accompanying 2nd person pronoun or name, can also be use for the vocative function, which might be an instance of ellipsis -- vocative(come, Hey).

For these reasons, I would be inclined to align myself with the practice in Sanskrit.
vocative(come, Joe)
case(Joe, Hey)

Note: The Sanskrit, above, has a translation ‘oh, Sir’. For me, the use of the word “oh” introduces a falling tone in the word “Sir”, which is pretty much identical to the falling tone used when calling to someone way of in distance. The word “hey” does not seem to allow for a collocated tone like this.
It also seems that the word “oh” might collate with first, second and third person pronouns.

Hey, @Stormur, @ftyers, @jonorthwash ,@flammie, @OliverHellwig, @dan-zeman, @jasiewert, @garanes, @nikopartanen   is this food for thought with regard to the vocative, or has it simply been easier to ignore the vocative?

**Comment by dan-zeman** [comment]
> is this food for thought with regard to the vocative, or has it simply been easier to ignore the vocative?

It has been easier to ignore it, except for languages where it is a morphological case (because even in these languages its function seems to be very different from all other cases).

**Comment by amir-zeldes** [comment]
> has it simply been easier to ignore the vocative?

I don't think we're ignoring it, it has a deprel after all. It's just not a case as such in English, because no word has a unique form for it (but pronouns do distinguish Nom vs. Acc vs. Gen in English), and because unlike normal case dependents in English (i.e. prepositions), words like "hey" can appear by themselves. Compare:

  * Hey!
  * `*` among

In other words, "hey" does not follow the valency structure of English case dependents, and seems to be a different word class, probably more like sentential adverbials or interjections, which is why it has the deprel `discourse` in the English TBs.

**Comment by Stormur** [comment]
Yes, probably the most telling diagnostics is that _hey_ can be used independently, i.e. it is not bound (a clitic, in Haspelmathian terms), and no real ellipsis can be assumed. That is also why it is classified as `INTJ`, it is more of a cry than an actual lexical element.  

In general, I would not put `Case` on adpositions or similar elements anyway if there is no change in the form (of the adposition). The function of the phrase is already managed by the relation it receives, in this case `vocative`. For example, in Latin we consider _o_ (which is different from a surprise _oh_, compare modern Italian where the opening and length of the vowel change) a particle introducing a vocative, but we only label it with `PartType=Emp`(hatic), which is a lexical, not a morphosyntactic distinction.

**Comment by AngledLuffa** [comment]
Hey (see what I did there?)

wish I had thought of this *before* we were 1200 sentences into labeling xpos and features, but...

if making different types of INTJ a feature isn't ideal, would it be reasonable to put that information in the xpos instead?

**Comment by nschneid** [comment]
For GUM and EWT we adhere to Penn Treebank tags for XPOS. INTJ should correspond closely to `UH`.
